controllers/utils/volumegroupcontent.go

```diff
@@ -315,3 +314,32 @@ func UpdateVGCByResponse(client client.Client, vgc *volumegroupv1.VolumeGroupCon
 	}
 	return nil
 }
+
+func DeletePVCsUnderVGC(logger logr.Logger, client client.Client, vgc *volumegroupv1.VolumeGroupContent, driver string) error {
+	logger.Info(fmt.Sprintf(messages.DeletePVCsUnderVGC, vgc.Namespace, vgc.Name))
+	for _, pv := range vgc.Status.PVList {
+		pvcName := getPVCNameFromPV(pv)
+		pvcNamespace := getPVCNamespaceFromPV(pv)
+		if pvcNamespace == "" || pvcName == "" {
+			pvc, err := getMatchingPVCFromPVCListToPV(logger, client, pv.Name, driver)
+			if err != nil {
+				return err
+			}
+			if pvc.Name == "" || pvc.Namespace == "" {
+				logger.Info(fmt.Sprintf(messages.CannotFindMatchingPVCForPV, pv.Name))
+				continue
+			}
+			pvcName = pvc.Name
+			pvcNamespace = pvc.Namespace
+		}
```

Comment:
why is this? what is the scenario?

Reply:
for this is a possibility that someone can edit the `PV` `ClaimRef` field, it's for extra caution but it shouldn't happen a lot, do you think I should remove it and just relay on the `ClaimRef` field?

Reply:
no, it seems that a `PV` should be working as usual without the `ClaimRef`